cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -26,21 +26,31 @@ import (
 )
 
 const (
-	DefaultServiceDNSDomain   = "cluster.local"
-	DefaultServicesSubnet     = "10.96.0.0/12"
-	DefaultKubernetesVersion  = "stable-1.7"
-	DefaultAPIBindPort        = 6443
+	// DefaultServiceDNSDomain defines default cluster-internal domain name fot Services and Pods
+	DefaultServiceDNSDomain = "cluster.local"
+	// DefaultServicesSubnet defines default service subnet range
+	DefaultServicesSubnet = "10.96.0.0/12"
+	// DefaultKubernetesVersion defines default kubernetes version
+	DefaultKubernetesVersion = "stable-1.7"
+	// DefaultAPIBindPort defines default API port
+	DefaultAPIBindPort = 6443
+	// DefaultAuthorizationModes defines default authorization modes
 	DefaultAuthorizationModes = "Node,RBAC"
-	DefaultCACertPath         = "/etc/kubernetes/pki/ca.crt"
-	DefaultCertificatesDir    = "/etc/kubernetes/pki"
-	DefaultEtcdDataDir        = "/var/lib/etcd"
-	DefaultImageRepository    = "gcr.io/google_containers"
+	// DefaultCACertPath defines default location of CA certificate
+	DefaultCACertPath = "/etc/kubernetes/pki/ca.crt"
+	// DefaultCertificatesDir defines default certificate directory
+	DefaultCertificatesDir = "/etc/kubernetes/pki"
+	// DefaultEtcdDataDir defines default location of etcd
+	DefaultEtcdDataDir = "/var/lib/etcd"
+	// DefaultImageRepository defines default image registry
+	DefaultImageRepository = "gcr.io/google_containers"
 )
 
 func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
+// SetDefaults_MasterConfiguration assigns default values to Master node
 func SetDefaults_MasterConfiguration(obj *MasterConfiguration) {
```

Comment:
Does this not trigger a warning as well? Pretty sure function should not have '_'.

Reply:
As per discussion for api machinery folks, we cannot change name for "default" functions as they are used for code generation and '_'  in the name is the requirement.